tex/generic/pgf/libraries/pgflibrarycurvilinear.code.tex

```diff
@@ -106,7 +106,7 @@
   \pgf@curvilinear@length@a\pgfmathresult pt%
   \ifdim\pgf@curvilinear@length@a>1pt\relax%
     % Ok, too large, let us make this smaller
-    \pgfmathdivide@{\pgf@sys@tonumber\pgf@curvilinear@time@a}{\pgf@sys@tonumber\pgf@curvilinear@length@a}%
+    \pgfmathdivide@{\pgf@curvilinear@time@a}{\pgf@curvilinear@length@a}%
```

Comment:
I'm pretty sure that should not have been reverted.  This was part of a bugfix.

Reply:
See https://github.com/pgf-tikz/pgf/pull/711#issuecomment-513389257